src/sentry/sentry_metrics/indexer/id_generator.py

```diff
@@ -19,32 +20,32 @@ def reverse_bits(number: int, bit_size: int) -> int:
 
 # we will have room b/n version and time since for a while
 # so let's reverse the version bits to grow to the right
-# instead of left should we need more than 4bits for version
-#
-#  version             time since
-# (0100    | 00000001001000101000000111100011)
-#
-# (10000)  |  0000001001000101000000111100011)
+# instead of left should we need more than 3 bits for version
 
 _VERSION_PREFIX = reverse_bits(_VERSION, _VERSION_BITS)
 
 
 def get_id() -> int:
     """
     Generates IDs for use by indexer storages that do not have autoincrement sequences (e.g. CloudSpanner).
+
     This function does not provide any guarantee of uniqueness, just a low probability of collisions.
     It relies on the database to be strongly consistent and reject writes with duplicate IDs. These should
     be retried with a newly generated ID.
 
     The ID generated is in roughly incrementing order.
+
+    Metric IDs are 64 bit but this function only generates IDs that fit in 63 bits. The leading bit is always zero.
+    This is because they were stored in Postgres as BigInt (signed 64 bit) and we do not want to change that now.
+    In ClickHouse it is an unsigned 64 bit integer.
     """
 
     now = int(time.time())
     time_since_epoch = now - _EPOCH_START
```

Comment:
We should probably rename `_EPOCH_START` to something like `_SPANNER_INDEXER_EPOCH_START`. The current one suggests that it is from linux epoch start

Reply:
I can add the _INDEXER prefix. It's probably not going to just be for spanner so i'll omit that part